crates/turborepo-lib/src/process/child.rs

```diff
@@ -177,6 +179,18 @@ impl Child {
     /// Start a child process, returning a handle that can be used to interact
     /// with it. The command will be started immediately.
     pub fn spawn(mut command: Command, shutdown_style: ShutdownStyle) -> io::Result<Self> {
+        let label = {
+            let cmd = command.as_std();
+            format!(
+                "({}) {} {}",
+                cmd.get_current_dir()
+                    .map(|dir| dir.to_string_lossy())
+                    .unwrap_or_default(),
+                cmd.get_program().to_string_lossy(),
+                cmd.get_args().map(|s| s.to_string_lossy()).join(" ")
+            )
```

Comment:
This is a [port from Go](https://github.com/vercel/turbo/blob/main/cli/internal/process/child.go#L112) so we can match outputs. Note that Go includes the program in `Args` while Rust does not so we must add it explicitly